### PR TITLE
create the skeleton for the public notebook.

### PR DESCRIPTION
**Pull request by bouzaghrane**
Starting this pull request with creating the skeleton for my public notebook.

### CODE REVIEW

**Comment by bouzaghrane** [comment]
@timothyb0912 Any advice on how to reorganize code within the simulation loop? It is way too long and I think it would be better if we can break it down to separate components outside of the loop for easier editing by users if need be? What's the smoothest way of doing this?

**Comment by timothyb0912** [comment]
@bouzaghrane Here are my first comments off an incomplete look at your block of simulation code. I think you should define the major collections of steps that are being performed. I'm thinking over-arching steps like "simulate new data", "estimate new models", "perturb variables", "store probabilities." I would then collect the logic for these steps in classes (try to identify relevant design patterns and use them here). In the for loop, I would essentially just create and use these objects via a single overall function that performs the desired task. I would likely just overwrite / populate the __call__ method.

File: notebooks/final/7.0-mab-selection-on-observables-final.md
Comment (timothyb0912) [inline notebooks/final/7.0-mab-selection-on-observables-final.md:22]: nit: Can you put each sentence on a separate line? It makes git diffs more meaningful as it'll allow highlighting just the part of the sentence that changed instead of the whole line containing unchanged sentences.

I think that the purpose of the notebook is to show if one ignores the structure of one's causal graph (i.e., one's treatment assignment mechanism), one can compute heavily biased treatment effect estimates. We contrast treatment effect estimation while using a correct causal graph vs using the naive, all-variables-are-independent default causal graph.

File: notebooks/final/7.0-mab-selection-on-observables-final.md
Comment (timothyb0912) [inline notebooks/final/7.0-mab-selection-on-observables-final.md]:
```suggestion
This notebook uses the dataset and the MNL utility specification from [Brathwaite and Walker (2016)](https://arxiv.org/abs/1606.05900) for demonstration.
```

File: notebooks/final/7.0-mab-selection-on-observables-final.md
Comment (timothyb0912) [inline notebooks/final/7.0-mab-selection-on-observables-final.md]:
```suggestion
 - Defining different causal graphs representing different views for how individuals make mode choices. These causal graphs are based on the MNL utility functions from [Brathwaite and Walker (2016)](https://arxiv.org/abs/1606.05900)
```

File: notebooks/final/7.0-mab-selection-on-observables-final.md
Comment (timothyb0912) [inline notebooks/final/7.0-mab-selection-on-observables-final.md]:
```suggestion
 - Perturbing one of the variables (e.g.: Travel Distance) to simulate a policy intervention.
```

File: notebooks/final/7.0-mab-selection-on-observables-final.md
Comment (timothyb0912) [inline notebooks/final/7.0-mab-selection-on-observables-final.md]:
```suggestion
 - Calculating and plotting the distributions of treatment effects according to different causal graphs. 
```

**Review by timothyb0912 — CHANGES_REQUESTED** [review]
I'm not done reviewing yet! I'm just putting a placeholder for the moment.

File: src/observables/distfit.py
Comment (timothyb0912) [inline src/observables/distfit.py]: The `verbose=True` specification gives me the following error when I try to run your public notebook on my computer.
Screen Shot 2020-11-07 at 3 31 38 PM

I remember you said you had to do something to fitter to get this to work? Is there anyway we can do without choices like that? It makes it impossible for our work to be reproduced.

Comment (bouzaghrane) [reply src/observables/distfit.py]: Just fixed it in the new version. It seems like a newer version of the fitter package gets rid of this verbose argument.

**Comment by timothyb0912** [comment]
Let's merge this if there are no more **immediate** changes you wanna make!

**Review by timothyb0912 — APPROVED** [review]

**Comment by timothyb0912** [comment]
🔒 Wait actually, before you do any merging...

Can we pare down the number of affected files in this merge? You're just adding your public notebook and the files in `src` that support it right?

Can you checkout the version of all other files from master and only retain changes to the smallest possible number of files?

**Review by timothyb0912 — CHANGES_REQUESTED** [review]
Please pare down changes to a minimal set of files.

**Comment by bouzaghrane** [comment]
@timothyb0912 okay I'm not entirely sure how to merge just specific files and didn't want to mess this up. I know which files need to be merged and we can talk about merging them tomorrow.

**Comment by timothyb0912** [comment]
@bouzaghrane I take it this can be closed right?

**Comment by bouzaghrane** [comment]
Yup! @timothyb0912 closing this.

**Pull request closed without merging**
